var/exp_repo/experiments/caliper/experiment.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Caliper(Experiment):
```

Comment:
I think the failure at https://github.com/LLNL/benchpark/actions/runs/11446987606/job/31847369138?pr=356 will be resolved if you make it so that `caliper` does not inherit `Experiment` (which I think is what you wanted to do). If that doesn't work I'll likely have to investigate it (but that failure in the dry runs does in fact look legitimate, so we'll want to address it before merging this).

Reply:
This coulda also be addressed by undoing https://github.com/LLNL/benchpark/pull/356/commits/2a0a077d6f85ee4861e0a7bf07290a96ca3bf138 I think